cli/eth.go

```diff
@@ -137,6 +139,56 @@ var EthCallSimulateCmd = &cli.Command{
 	},
 }
 
+var EthGetContractAddress = &cli.Command{
+	Name:      "contract-address",
+	Usage:     "Generate contract address from smart contract code",
+	ArgsUsage: "[senderEthAddr] [salt] [code binary]",
```

Comment:
Do I actually have to input 1s and 0s? (Sorry for putting sarcasm in text, but there is probably a better name for this field). Looks like it's actually a path to a file with hex?

Reply:
Yes. "Contract bin file"?
is there a way I can describe the params as part of the cli? That would be more helpful to the user

Reply:
I think code binary path is fine then

Reply:
it is. im confused by your confusion. Are you saying hex files are not binaries?

Reply:
You're right. I'm dumb lol

Reply:
Wait, nevermind. Changed my mind again. Will come to the office to bikeshed this in ~15 mins lol